examples/test-controls/index.html

```diff
@@ -11,7 +11,7 @@
     <a-scene stats="true">
       <!-- We wrap the camera in an <a-entity> so we can position it without colliding with the HMD updates -->
       <a-entity>
-        <a-entity position="0 1.8 4" camera look-controls wasd-controls></a-entity>
+        <a-entity position="0 1.8 4" camera controls></a-entity>
```

Comment:
controls does not mean much in a general way. It's application dependent. FPS vs Panorama Viewer vs Platform Game vs Flight simulator. It's not the same controlling a camera, than a character in 3rd person view. I suggest them giving a prefix that convey what kind of controls we're talking about. Relying on documentation I think it's just lazy thinking on naming things. People don't read docs.